indexer-app/resources/schemas/index/v0.0.1/schema.json

```diff
@@ -9,6 +9,12 @@
         "MetadataSpecification": {
             "description": "Contains several fields to define the specifics of the grid",
             "$ref": "#/definitions/MetadataSpecificationType"},
+        "SubConceptType": {
+          "type": "string",
+          "maxItems": 1,
```

Comment:
maxItems doesn't mean anything here.  This isn't an array.  Please remove it.

Reply:
I think I meant min count, how about that?

Reply:
you already have minLength: 1.